2022-03-25 23:06:05,102 INFO    MainThread:42329 [wandb_setup.py:_flush():75] Loading settings from /nethome/mrungta8/.config/wandb/settings
2022-03-25 23:06:05,102 INFO    MainThread:42329 [wandb_setup.py:_flush():75] Loading settings from /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning_example_creation/wandb/settings
2022-03-25 23:06:05,102 INFO    MainThread:42329 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-25 23:06:05,103 INFO    MainThread:42329 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'Mukund/RE_improved_baseline/meta_learning_example_creation/train_tacred.py', 'program': 'train_tacred.py'}
2022-03-25 23:06:05,103 INFO    MainThread:42329 [wandb_init.py:_log_setup():386] Logging user logs to /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning_example_creation/wandb/run-20220325_230605-1btw0ldt/logs/debug.log
2022-03-25 23:06:05,103 INFO    MainThread:42329 [wandb_init.py:_log_setup():387] Logging internal logs to /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning_example_creation/wandb/run-20220325_230605-1btw0ldt/logs/debug-internal.log
2022-03-25 23:06:05,103 INFO    MainThread:42329 [wandb_init.py:init():420] calling init triggers
2022-03-25 23:06:05,103 INFO    MainThread:42329 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-03-25 23:06:05,103 INFO    MainThread:42329 [wandb_init.py:init():471] starting backend
2022-03-25 23:06:05,103 INFO    MainThread:42329 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-25 23:06:05,119 INFO    MainThread:42329 [backend.py:ensure_launched():219] starting backend process...
2022-03-25 23:06:05,133 INFO    MainThread:42329 [backend.py:ensure_launched():225] started backend process with pid: 42472
2022-03-25 23:06:05,135 INFO    MainThread:42329 [wandb_init.py:init():480] backend started and connected
2022-03-25 23:06:05,148 INFO    MainThread:42329 [wandb_init.py:init():550] updated telemetry
2022-03-25 23:06:05,188 INFO    MainThread:42329 [wandb_init.py:init():581] communicating current version
2022-03-25 23:06:06,249 INFO    MainThread:42329 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-25 23:06:06,249 INFO    MainThread:42329 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-25 23:06:06,664 INFO    MainThread:42329 [wandb_init.py:init():624] starting run threads in backend
2022-03-25 23:06:08,493 INFO    MainThread:42329 [wandb_run.py:_console_start():1827] atexit reg
2022-03-25 23:06:08,494 INFO    MainThread:42329 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2022-03-25 23:06:08,496 INFO    MainThread:42329 [wandb_run.py:_redirect():1706] Redirecting console.
2022-03-25 23:06:08,506 INFO    MainThread:42329 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-25 23:06:08,507 INFO    MainThread:42329 [wandb_init.py:init():651] run started, returning control to user process
2022-03-25 23:21:43,795 INFO    MainThread:42329 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 1
2022-03-25 23:21:43,799 INFO    MainThread:42329 [wandb_run.py:_restore():1769] restore
2022-03-25 23:21:46,724 INFO    MainThread:42329 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 985
  total_bytes: 985
}

2022-03-25 23:21:46,947 INFO    MainThread:42329 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 985
  total_bytes: 985
}

2022-03-25 23:21:47,216 INFO    MainThread:42329 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 985
  total_bytes: 985
}

2022-03-25 23:21:47,320 INFO    MainThread:42329 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 985
  total_bytes: 9577
}

2022-03-25 23:21:47,424 INFO    MainThread:42329 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9577
  total_bytes: 9577
}

2022-03-25 23:21:47,528 INFO    MainThread:42329 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9577
  total_bytes: 9577
}

2022-03-25 23:21:47,632 INFO    MainThread:42329 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9577
  total_bytes: 9577
}

2022-03-25 23:21:47,736 INFO    MainThread:42329 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9577
  total_bytes: 9577
}

2022-03-25 23:21:47,840 INFO    MainThread:42329 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9577
  total_bytes: 9577
}

2022-03-25 23:21:48,127 INFO    MainThread:42329 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9577
  total_bytes: 9577
}

2022-03-25 23:21:48,470 INFO    MainThread:42329 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9577
  total_bytes: 9577
}
local_info {
}

2022-03-25 23:21:49,971 INFO    MainThread:42329 [wandb_run.py:_append_files():2194] logging synced files
